core-bundle/src/Resources/functions/symfony-polyfill.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Symfony\Component\HttpFoundation;
```

Comment:
I thought so, too, but it seems we were both wrong: https://3v4l.org/qApgY

Reply:
It is even documented: https://www.php.net/manual/en/language.namespaces.definitionmultiple.php 😄

Reply:
The documentation says:

> This syntax is not recommended for combining namespaces into a single file. Instead it is recommended to use the alternate bracketed syntax.

So I guess we should use brackets here.

Reply:
We're doing the same when emitting PHP from the token parser btw.

Reply:
It is different there IMHO, because we also have code in the global namespace and thus the bracketed syntax is technically required. But I have adjusted the polyfill in bc33c0b9c881a7baad52fffda5facd5f40becc15.